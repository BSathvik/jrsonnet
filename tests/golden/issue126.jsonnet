{
	int_lhs: 1 << 32,
	unary_plus: +1,
}
