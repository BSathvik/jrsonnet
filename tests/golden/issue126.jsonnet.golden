{
    "int_lhs": 4294967296,
    "unary_plus": 1
}